src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/services/PDFGenerator.java

```diff
@@ -258,7 +280,7 @@ private void renderGroup(FormLayoutElement element, boolean childGroup) throws I
           }
         }
         if (childElement.getType().equalsIgnoreCase("group")) {
-          renderGroup(childElement, true);
+            renderRepeatingGroup(childElement, false);
```

Comment:
```suggestion
            renderRepeatingGroup(childElement, true);
```

Should be true here, right?

Reply:
Oh yess :) Nice catch!